analyticsclient/client.py

```diff
@@ -114,7 +148,17 @@ def _request(self, resource, timeout=None, data_format=DF.JSON):
 
         try:
             uri = '{0}/{1}'.format(self.base_url, resource)
-            response = requests.get(uri, headers=headers, timeout=timeout)
+
+            if method == self.METHOD_GET:
+                response = requests.get(uri, headers=headers, timeout=timeout)
+            elif method == self.METHOD_POST:
+                response = requests.post(uri, data=post_data, headers=headers, timeout=timeout)
+            else:
+                raise ValueError(
+                    'Client._request: invalid value of method: expected {0} or {1}'.format(
```

Comment:
Nit: great message -- might be improved by saying what the invalid value is too.